.github/workflows/test-kitchen.yml

```diff
@@ -1,40 +1,17 @@
 ---
-name: 'Lint & Unit Test'
+name: 'Test'
 
 'on':
   pull_request:
   push:
     branches: [main]
 
 jobs:
-  delivery:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Check out code
-        uses: actions/checkout@v2
-      - name: Run Chef Delivery
-        uses: actionshub/chef-delivery@main
-        env:
-          CHEF_LICENSE: accept-no-persist
-
-  yamllint:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Check out code
-        uses: actions/checkout@v2
-      - name: Run yaml Lint
-        uses: actionshub/yamllint@main
-
-  mdl:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Check out code
-        uses: actions/checkout@v2
-      - name: Run Markdown Lint
-        uses: actionshub/markdownlint@main
+  lint-unit:
+    uses: sous-chefs/.github/.github/workflows/lint-unit.yml@c4d27f78aaae5e927289b1d7f0c4094ba8cf2846
```

Comment:
Not 100% sold on this location.

Reply:
Why does it need to use a hash here instead of a branch name?

Reply:
Happy for it to be either. This was just showing that you could do either 👍🏼

Reply:
Fun fact: I'd also place it in the exact same directory for our tecRacer cookbooks etc..
So my opinion: before creating a new repo only for those reusable workflows, which probably could become an option in the future to be stored in that directory, I'd place them there as well :-)